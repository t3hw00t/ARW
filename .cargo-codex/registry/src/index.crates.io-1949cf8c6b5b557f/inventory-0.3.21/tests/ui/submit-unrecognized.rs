pub struct Thing;

inventory::submit!(Thing);

fn main() {}
