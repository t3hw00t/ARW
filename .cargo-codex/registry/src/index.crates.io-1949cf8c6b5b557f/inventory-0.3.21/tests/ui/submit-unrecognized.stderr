error[E0277]: the trait bound `Thing: ErasedNode` is not satisfied
 --> tests/ui/submit-unrecognized.rs:3:1
  |
3 | inventory::submit!(Thing);
  | ^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Collect` is not implemented for `Thing`
  |
  = note: required for `Thing` to implement `ErasedNode`
  = note: required for the cast from `&Thing` to `&'static (dyn ErasedNode + 'static)`
  = note: this error originates in the macro `$crate::__do_submit` which comes from the expansion of the macro `inventory::submit` (in Nightly builds, run with -Z macro-backtrace for more info)
